litefs/cloud-backups.html.md.erb

```diff
@@ -0,0 +1,82 @@
+---
+title: Using LiteFS Cloud for Backups
+layout: docs
+sitemap: false
+nav: litefs
+toc: true
+---
+
+The simplest way to implement disaster recovery for your LiteFS cluster is
+to use Fly.io's LiteFS Cloud for backups. LiteFS Cloud provides reliable backup
+and restore functionality, which reduces the risk of of data loss in the event
+that your cluster is lost. You'll also be able to restore your cluster to any
+point in time (not just the most recent snapshot), in case you need to revert
+your state.
+
+In this doc, we'll show you how to add LiteFS Cloud to an existing LiteFS
+cluster. If you're not using LiteFS yet, take a look at the
+[getting started guide][], which walks you through setting LiteFS up for your
+app, including LiteFS Cloud.
+
+If you'd rather implement your own backups, take a look at the
+[Backing up your LiteFS cluster][] docs.
+
+[getting started guide]: /docs/litefs/getting-started
+[Backing up your LiteFS cluster]: /docs/litefs/backup
+
+## Creating a LiteFS Cloud cluster
+
+If you want to back up your database using LiteFS Cloud, you'll need to create
+a LiteFS Cloud cluster. You can do this in the Fly.io dashboard. First,
+[sign up for an account][Sign up] if you haven't already. After you're signed in,
+navigate to the LiteFS Cloud section on the left navbar, and then use the Create
+button to create a new cluster.
+[Here's a handy link to the LiteFS Cloud dashboard][LiteFS Cloud Dashboard] if you prefer!
+
+When you create your LiteFS Cloud cluster, you'll be asked to save an auth token.
+Keep this handy - you'll use it below.
+
+[Sign up]: /docs/hands-on/sign-up
+[LiteFS Cloud Dashboard]: https://fly.io/dashboard/personal/litefs
+
+## Configuring LiteFS to use LiteFS Cloud
+
+### Update LiteFS
+
+Before configuring LiteFS Cloud, you must update your LiteFS to a version that supports
+LiteFS Cloud. Update to `v0.5.0` or greater. You can do this by
+[installing the latest version][LiteFS Installation].
+
+[LiteFS Installation]: /docs/litefs/getting-started/#installing-litefs
```

Comment:
This link is a 404 now. Maybe point it at the Fly version?

Reply:
Ah thanks for catching this! Yes I will do that.